SUSE Dark Theme

richard-cox opened the pull request:
- apply custom dark theming to helm chart list chart card
- create `dark` theme in suse theme,
- dark suse theme only needed two changes to normal suse theme
  - core colour theme created with `mat-dark-theme` mixin
  - custom side nav colours
- ~~blocked on #404 #406~~

nwmac committed:
WIP: Suse Theme updated for new brand

nwmac committed:
Final updates. Logo, login screen

richard-cox committed:
Merge remote-tracking branch 'upstream/master'

richard-cox committed:
Fixes after merge

richard-cox committed:
Merge remote-tracking branch 'origin/suse-rebrand' into dark-mode

richard-cox committed:
Add SUSE dark theme
- apply custom dark theming to helm chart list chart card
- create `dark` theme in suse theme,
- dark suse theme only needed to changes
  - core colour theme created with `mat-dark-theme` mixin
  - custom side nav colours

richard-cox committed:
Fix build

richard-cox committed:
Merge remote-tracking branch 'origin/merge-upstream' into dark-mode

nwmac commented on src/frontend/packages/suse-extensions/src/custom/helm/list-types/monocular-chart-card/monocular-chart-card.component.theme.scss:
```diff
@@ -0,0 +1,37 @@
+  @if $is-dark == true {
+    $border-color: $black;
+    $background-color-top: lighten(#3f3f3f, 10);
+    $background-color-bottom: #3f3f3f;
```
Is this is available on $app-theme of $theme ?

richard-cox committed:
Light & Dark mode changes following review
- dark mode - update link colour
- dark mode - update colour scheme, including primary dark mode green
- dark mode - update page side nav active colours
- dark mode - remove border betwen side nav and content
- dark mode - make side nav background darker
- light mode - update standard colour pallet given suse primary colour

richard-cox committed:
Merge remote-tracking branch 'origin/master' into dark-mode

# Conflicts:
#	docs/customizing.md
#	package-lock.json
#	src/frontend/packages/suse-extensions/package.json
#	src/frontend/packages/suse-extensions/src/custom/suse-login/suse-login.component.html
#	src/frontend/packages/suse-extensions/src/custom/suse-login/suse-login.component.scss
#	src/frontend/packages/suse-theme/_index.scss
#	src/frontend/packages/suse-theme/package.json
#	src/frontend/packages/suse-theme/sass/colors.scss
#	src/frontend/packages/suse-theme/sass/suse.scss
#	src/jetstream/go.sum
#	src/tsconfig.json

nwmac reviewed (approved):
LGTM
